chef/cookbooks/crowbar/templates/default/crowbar-migrate-keystone-and-start.sh.erb

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# This makes sure openstack related services are really shut down
```

Comment:
You might wanna edit these copied-and-pasted content (same for the other script)